index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome books</title>
+    <link rel="stylesheet" href="/src/style.css">
+  </head>
+  <body>
+    <h1>Awesome books</h1>
+    <ul id="books"></ul>
+    <form action="#" id="form">
+      <input type="text" name="title" id="title" placeholder="Title" />
+      <br />
+      <br />
+      <input type="text" name="author" id="author" placeholder="Author" />
+      <br />
+      <br />
+      <input
+        type="submit"
+        name="submit-button"
+        id="submit-button"
+        value="Add book"
+      />
+    </form>
+
+    <!-- SCRIPTS -->
+    <script src="/src/main.js"></script>
+    <script>
+      function newBook(e) {
+        e.preventDefault();
+        title = document.getElementById("title");
+        author = document.getElementById("author");
+        myBooks.push(new Book(title.value, author.value));
+        updateLocalStorage();
+        title.value = "";
+        author.value = "";
+      }
+      const form = document.getElementById("form");
+      form.addEventListener("submit", newBook);
+    </script>
+    <script>
+      function getBooks() {
+        myBooks = JSON.parse(localStorage.myBooks);
+        books = document.getElementById("books");
+        books.innerHTML = "";
+        let id = 0;
+        for (book of myBooks) {
+          books.innerHTML += `
+          <li>
+            <p>${book.title}</p>
+            <p>${book.author}</p>
+            <button onClick="destroyBook(${id++})">Remove</button>
+          </li>`;
+        }
+      }
+      if (localStorage.length > 0) getBooks();
+    </script>
```

Comment:
It would be nice if we separate the Js code from our HTML file following JS best practice. 👌🏼 
![1680250e-f7de-45eb-9ed7-2f6cae7e7463](https://user-images.githubusercontent.com/24830039/128780222-c8358311-fbfe-460b-8ad2-b52483350c51.png)